packages/react/src/components/Popover/index.scss

```diff
@@ -0,0 +1,11 @@
+.c__popover {
+  position: absolute;
+  display: flex;
+  width: fit-content;
+  z-index: 1;
+
+  &:not(&--borderless) {
+    box-shadow: 0 1px 4px rgba(0, 0, 0, 0.3);
```

Comment:
What do you think to create a sass placeholder in order to reuse the same box shadow everywhere ?